extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -109,11 +145,14 @@ public function buildEditPackage( $oPage, $oUser, $oRevision = null, $oLocalFile
 		$this->setMediaLinks( $oPage );
 		$this->setTotalWords( str_word_count( $rev_text ) );
 
-		if ( $oLocalFile instanceof File ) {
-			$this->setMediaType( $oLocalFile );
-			$this->setIsImageForReview( ImagesService::isLocalImage( $oTitle ) );
-		} else {
-			$this->setIsImageForReview( false );
+		if ( $oTitle instanceof Title && $oTitle->getNamespace() == NS_FILE ) {
```

Comment:
Please use `Title::inNamespace` method instead of comparsion.
Do you expect `$oTitle` to be something else or just checks if it was created? If second maybe simple check is enough:
`if ( $oTitle && $oTitle->inNamespace(NS_FILE) {`